chains/types.ts

```diff
@@ -1,27 +1,337 @@
+  | 'logion-testnet-standalone-2'
+  | 'mandala-testnet'
+  | 'manta-testnet'
+  | 'minix-testnet'
+  | 'moonbase-alpha-testnet'
+  | 'moonshadow-testnet'
+  | 'mybank-testnet'
+  | 'myriad-testnet'
+  | 'nftmart-testnet'
+  | 'nodle-testnet'
+  | 'opal-testnet'
+  | 'opportunity-testnet'
+  | 'pangolin-testnet'
+  | 'pangoro-testnet-standalone'
+  | 'pangoro-testnet-westend'
+  | 'phala-testnet'
+  | 'phoenix-testnet'
+  | 'pichiu-testnet'
+  | 'polkadex-testnet'
+  | 'polymesh-testnet'
+  | 'pontem-testnet'
+  | 'prism-testnet'
+  | 'realis-testnet'
+  | 'riochain-testnet'
+  | 'robonomics-testnet'
+  | 'rocfinity-testnet'
+  | 'rococo-acurast-testnet'
+  | 'rococo-amplitude-testnet'
+  | 'rococo-aventus-testnet'
+  | 'rococo-bajun-testnet'
+  | 'rococo-basilisk-testnet'
+  | 'rococo-bifrost-testnet'
+  | 'rococo-bitgreen-testnet'
+  | 'rococo-bridgehub-testnet'
+  | 'rococo-catalyst-testnet'
+  | 'rococo-confti-testnet'
+  | 'rococo-contracts-testnet'
+  | 'rococo-crust-testnet'
+  | 'rococo-dali-testnet'
+  | 'rococo-dolphin-testnet'
+  | 'rococo-ethos-testnet'
+  | 'rococo-frequency-testnet'
+  | 'rococo-genshiro-testnet'
+  | 'rococo-hydra-dx-testnet'
+  | 'rococo-imbue-testnet'
+  | 'rococo-integritee-testnet'
+  | 'rococo-kabocha-testnet'
+  | 'rococo-kilt-testnet'
+  | 'rococo-litentry-testnet'
+  | 'rococo-mangata-testnet'
+  | 'rococo-md-5-testnet'
+  | 'rococo-moonsama-testnet'
+  | 'rococo-nodle-testnet'
+  | 'rococo-origin-trail-parachain-testnet'
+  | 'rococo-pangolin-testnet'
+  | 'rococo-sora-testnet'
+  | 'rococo-spreehafen-testnet'
+  | 'rococo-statemint-testnet'
+  | 'rococo-subsocial-testnet'
+  | 'rococo-subzero-testnet'
+  | 'rococo-testnet'
+  | 'rococo-tinkernet-testnet'
+  | 'rococo-turing-testnet'
+  | 'rococo-unit-network-testnet'
+  | 'rococo-virto-testnet'
+  | 'rococo-watr-testnet'
+  | 'rococo-zeitgeist-testnet'
+  | 'sherpax-testnet'
+  | 'shibuya-testnet'
+  | 'skyekiwi-testnet'
+  | 'snowbridge-testnet'
+  | 'soonsocial-testnet'
+  | 'sora-substrate-testnet'
+  | 'stagex-testnet'
+  | 'subdao-testnet'
+  | 'subgame-testnet'
+  | 'subspace-farmnet-testnet'
+  | 'subspace-gemini-1-testnet'
+  | 'subspace-gemini-2-a-testnet'
+  | 'subspace-testnet'
+  | 'substrate-testnet'
+  | 'sumi-staging-testnet'
+  | 't-0-rn-testnet'
+  | 'ternoa-alphanet-testnet'
+  | 'ternoa-testnet'
+  | 'thebifrost-testnet'
+  | 'uniarts-testnet'
+  | 'unique-testnet'
+  | 'unitv-testnet'
+  | 'vara-testnet'
+  | 'vodka-testnet'
+  | 'web-3-games-testnet'
+  | 'westend-bridge-hub-testnet'
+  | 'westend-collectives-testnet'
+  | 'westend-pichiu-testnet'
+  | 'westend-standard-testnet'
+  | 'westend-testnet'
+  | 'westmint-testnet'
+  | 'whala-testnet'
+  | 'z-cloak-testnet'
+  | 'zeitgeist-testnet'
+  | 'zero-testnet';
+
+export type Account = '*25519' | 'secp256k1' | 'Sr25519';
+
+export type JsonString = string;
+
+export interface Token {
+  symbol: string;
+  decimals: number;
+  ed: string;
+  onChainId: JsonString | number;
```

Comment:
Similar to `ed`, this is not self explanatory to me. 
We need to start with the docs anyways, we could right away already adding some JSDoc annotations to the codebase.

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
it is the `existentialDeposit`. I renamed it and added a comment